### DISCUSSION TITLE
A couple of suggestions.

### DISCUSSION THREAD

**Discussion by hbkb001** [xspanger3770/GlobalQuake#126] [Ideas]
Hello,
Love this app / program the Visual map / globe is really nice.
the first suggestion is to align the timing closer to the reporting agency's reports as it is a bit confusing (Yes I use Geonet to double check things)
the second, maybe look at putting the quake ID in the information reported so we can track down the official report of the quake if we want / need to do so.

apart from that keep up the good work and keep improving it!
Cheers from NZ

**Comment by xspanger3770** [comment]
Hello!

Please keep in mind that GlobalQuake locates all the earthquakes by itself, including the location, estimated magnitude and origin time. All the reports are very preliminary and should only be used for entertainment purposes. GlobalQuake doesn't fetch the earthquake data from any reporting agency, and thus cannot provide any quake ID information.

Thanks for your suggestion and feel free to ask more!
Jakub.

**Reply by hbkb001** [reply]
Hey Jakub, Thanks for the response.
When setting the Home position for the cinema mode I've found it doesn't lock to the entered co-ordinates ... is there a possibility of either locking it to the co-ordinates entered or by country? so living in NZ could select "New Zealand" from a list and have it lock onto that country as the "home position" ... otherwise Loved the update to 0.9.6

**Reply by xspanger3770** [reply]
Hi! 

The cinema mode prefers to lock on active earthquakes on the map, but if there are none, it will zoom into areas with potential shaking detected. If there are none of these, then it will lock to your home position.
In 0.9.6, the cinema mode is very early feature and will be improved in the future.

Have a nice day!
